CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
 # Changelog
 
+## [1.56](https://github.com/brave/brave-ios/releases/tag/v1.56)
+
+### Web3
+
+- Implemented “Select a token to send” view which allows selecting an account and an asset at the same time. ([#7523](https://github.com/brave/brave-ios/issues/7523))
+- Removed network name from the “Public Encryption Key Request” panel. ([#7495](https://github.com/brave/brave-ios/issues/7495))
```

Comment:
Move it below Updated items

Also suggested change: `Removed network name from "Public Encryption Key Request" confirmation screen`